roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -275,6 +275,10 @@ spec:
             # Enable or disable usage report
             - name: FELIX_USAGEREPORTINGENABLED
               value: "{{ calico_usage_reporting }}"
+{% if calico_version is version('3.29.0', '>=') %}
```

Comment:
Is v3.29.0 rather than 3.29.0? Can you ensure this?

Reply:
Wydm? We had removed the v-prefix in #11890, I think we don't need compare with v?